src/components/Cover.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../styles/index.scss';
+import Pathways from './Pathways.jsx';
+
+
+function Cover() {
+  return (
+    <section class="cover">
+      <h1>The most comprehensive artists catalogue</h1>
+        <Pathways />
```

Comment:
Indentación extra que puede eliminarse:
```suggestion
  return (
    <section class="cover">
      <h1>The most comprehensive artists catalogue</h1>
      <Pathways />
```